vic/drivers/image/include/vic_driver_image.h

```diff
@@ -31,11 +31,12 @@
 
 #define VIC_DRIVER "Image"
 
+void display_current_settings(int);
 void get_forcing_file_info(param_set_struct *param_set, size_t file_num);
 void get_global_param(FILE *);
+void vic_force(void);
```

Comment:
Here I simply ordered the functions alphabetically, so vic_force() changed position.